man/man5/puppet.conf.5

```diff
@@ -30,7 +30,7 @@ The Puppet executables will ignore any setting that isn\'t relevant to their fun
 .IP "" 0
 .
 .P
-See the configuration guide \fIhttp://docs\.puppetlabs\.com/guides/configuring\.html\fR for more details\.
+See the configuration guide \fIhttps://docs\.puppetlabs\.com/guides/configuring\.html\fR for more details\.
```

Comment:
This link now points to http://docs.puppetlabs.com/puppet/latest/reference/config_about_settings.html